jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -112,6 +120,8 @@ define([
                     if ($helpArea) {
                         // remove if it exists since layout-specific help text will be inserted
                         $helpArea.remove();
```

Comment:
I ran that fiddle and got this result:

![image](https://cloud.githubusercontent.com/assets/6362426/15192734/20f5b028-1780-11e6-8009-fc63a3bb9256.png)